src/main/java/frc/robot/subsystems/SwerveModule.java

```diff
@@ -167,7 +176,9 @@ public void setDesiredState(SwerveModuleState desiredState) {
         desiredState = SwerveModuleState.optimize(desiredState, getState().angle);
         driveMotor.set(desiredState.speedMetersPerSecond / SwerveSubsystem.kPhysicalMaxSpeed);
         turningSetpoint = calculateSetpoint(desiredState.angle.getRadians());
-        turningPIDController.setReference(turningSetpoint, ControlType.kPosition);
+        int errorCode = turningPIDController.setReference(turningSetpoint, ControlType.kPosition).value;
```

Comment:
definition of variable should occur at the top of a method

Reply:
done, variable is now defined beforehand